test/test_make_install_with_bench.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -e
+
+USAGE_STR="Usage: $(basename $0) <workspace_dir> <build_dir> <install_dir> <build_type:Release|Debug>"
+
+if [ "$4" == "" ]; then
+	echo $USAGE_STR
+	exit 1
+fi
+
+WORKSPACE=$(realpath $1)
+BUILD_DIR=$(realpath $2)
+INSTALL_DIR=$(realpath $3)
+BUILD_TYPE=$4
+
+if [ ! -f $WORKSPACE/README.md ]; then
+	echo "Incorrect <workspace_dir>: $WORKSPACE"
+	echo $USAGE_STR
+	exit 1
+fi
+
+if [ "$BUILD_DIR" == "$WORKSPACE" ]; then
+	echo "Incorrect <build_dir>: $BUILD_DIR"
+	echo $USAGE_STR
+	exit 1
+fi
+
+if [ ! -d $INSTALL_DIR ]; then
+	echo "Incorrect <install_dir> (it should exist): $INSTALL_DIR"
+	echo $USAGE_STR
+	exit 1
+fi
+
+if [ "$BUILD_TYPE" != "Release" -a "$BUILD_TYPE" != "Debug" ]; then
```

Comment:
Maybe I'm not aware of something - but why is it so important to verify it's either of these two? What if I use e.g. `RelWithDebInfo`...?